README.md

```diff
@@ -51,6 +50,48 @@ blocked:000000
 bug:fc2929
 ```
 
+#### Deleting Labels
+
+Sometimes you might want to delete some labels (likely Github's default labels) in the process of setting up a repo with your label config.
+
+You can add in the your label config file a line with `-delete:tag-name` to go and delete a specific label.
+
+For example:
+
+```
+-delete:wontfix
```

Comment:
it would be great to have a convenience option of something like `-delete:gh-default-labels`